addons/wowl/static/src/services/action_manager/action_manager.ts

```diff
@@ -285,6 +291,7 @@ function makeActionManager(env: OdooEnv): ActionManager {
       Component = controller.Component;
       mounted() {
         controllerStack = nextStack; // the controller is mounted, commit the new stack
+        env.services.menus.setCurrentMenu(action._menuId);
```

Comment:
what about triggering an event on the bus?

Reply:
yes, i'm not sure either on how to update the navbar....